modules/reactor/src/meshgenerators/AssemblyMeshGenerator.C

```diff
@@ -314,20 +353,51 @@ AssemblyMeshGenerator::AssemblyMeshGenerator(const InputParameters & parameters)
 
       declareMeshProperty("background_region_ids", _background_region_id);
       declareMeshProperty("duct_region_ids", _duct_region_ids);
+      declareMeshProperty("background_block_names", _background_block_name);
+      declareMeshProperty("duct_block_names", _duct_block_names);
+    }
+  }
+
+  // Remove outer pin sidesets created by PolygonConcentricCircleMeshGenerator
```

Comment:
this works. An alternative, and the currently more MOOSE-way, is to delete the sidesets after stitching in the patternXXGenerator. In fact I thought the stitching would do that by default?

Reply:
I don't believe so. In these examples these sidesets are still preserved after stitching in the patternXXGenerator